frontend/src/styles.css

```diff
@@ -176,27 +171,58 @@ fieldset > :not(.field-separator) {
   text-align: center;
 }
 
-mat-card.default {
-  width: 400px;
+.mat-card-form {
+  width: 420px;
+  padding-bottom: 0px;
+  margin-bottom: 0px;
 }
 
-.margin.default {
-  margin: 10px;
+.link-container-beneath-form {
+  width: 420px;
+}
+
+.outer-stepper {
+  width: 710px;
+}
+
+.inner-stepper {
+  width: 680px;
+}
+
+.same-line-input {
+  max-width: 185px;
+}
+.nextline-stepper .mat-step-label.mat-step-label {
+  text-align: center;
+  text-overflow: inherit;
+  white-space: normal;
```

Comment:
We try to avoid global CSS classes where possible. Instead, use TailwindCSS.

The reasons are: 
- When code in the HTML templates are removed, CSS classes are often forgotten. 
- It's hard to keep an overview over available CSS classes. Everybody will define their own classes, which is against the concept of consistent and aligned classes.
- Changes to a global class can introduce several side effects.
- We don't have to invent classnames that nobody understands.

A page that is worth to read: https://tailwindcss.com/docs/utility-first

TailwindCSS has the nice concept of custom styles: https://tailwindcss.com/docs/adding-custom-styles, which can be used for the definition of a specific width. If it doesn't make sense to use TailwindCSS, prefer to use component scoped CSS classes.

Reply:
I'll move the lesser-used classes to the inline utility styles, but the form styling would probably be a mistake to split up--it's used across 9 forms in 9 different files which would ideally be templated/compontentized but would probably require more effort than it's worth. We can also talk about this at the next weekly, but honestly, I'm concerned the current approach is causing more problems than it's solving (that is, without a visual design spec you'll contniue to see more inconsistancies in styling with utility styles than with reusable classes).

Reply:
OK--shifted everything inline except for the form class that gets used across so many files. I can move that inline if you prefer, but I think the class is more maintainable given the project creation flow will be updated/added to at some point and in the absense of a component the various forms are derived from it makes it easier to find them all. Or I can try to componentize everything, but that probably deserves a separate issue because it'd be structurally changing multiple parts of the site.

Reply:
> We can also talk about this at the next weekly, but honestly, I'm concerned the current approach is causing more problems than it's solving (that is, without a visual design spec you'll contniue to see more inconsistancies in styling with utility styles than with reusable classes).

This might be more suited for discussion during the Collaboration Manager planning, but I can offer some insights based on our experience. We initially adopted both global and component-based styles, but this approach wasn't effective for us. The primary issue was the continuous addition of new classes, while the existing ones were seldom reused.

Style consistencies have never been a big priority and will continue to be less relevant than functionality and stability. In order to really have a consistent style, we need someone who has the time to actively enforce a consistent style. Regardless of whether we use inline or global styles. But we don't have the capacity for that, and I'm also worried that it could slow us down in the development process.